src/server/getters/get-topics.ts

```diff
@@ -2,8 +2,8 @@ import * as Knex from "knex";
 import { Address } from "../../types";
 
 interface TopicsRow {
-  topic: string,
-  popularity: number
+  topic: string;
+  popularity: number;
 }
 
 type TopicsInfo = TopicsRow[];
```

Comment:
Should be `Array<TopicsRow>`.  Going to stop marking these, I think the reason these aren't picked up by the linter is the subdirectories aren't being linted.